src/main/java/org/ml_methods_group/algorithm/sddrar/SDDRARioHandler.java

```diff
@@ -0,0 +1,70 @@
+package org.ml_methods_group.algorithm.sddrar;
+
+import org.ml_methods_group.algorithm.sddrar.rules.RulePack;
+
+import java.io.*;
+import java.util.List;
+
+public class SDDRARioHandler {
+
+    public static final String RULES_FILE = "rules.data";
+    public static final String METRICS_FILE = "metrics.data";
+    private static final String DATA_FOLDER = "/sddrar_data/";
+
+    public static void dumpRulePack(RulePack pack) {
+        dump(pack, RULES_FILE);
+    }
+
+    public static RulePack loadRulePack() {
+        return load(RULES_FILE);
+    }
+
+    public static void dumpMetrics(List<String> metrics) {
+        dump(metrics, METRICS_FILE);
+    }
+
+    public static List<String> loadMetrics() {
+        return load(METRICS_FILE);
+    }
+
+    public static void dumpDataSet(DataSet dataSet, String name) {
+        dump(dataSet, name);
+    }
+
+    public static DataSet loadDataSet(String name) {
+        return load(name);
+    }
+
+    public static void dump(Object obj, String name) {
+        try {
+            File file = new File(System.getProperty("user.home") + "/ArchitectureReloaded/sddrar_data/" + name);
```

Comment:
This method is required only for train SDDRAR and should not be used in production. I think it will be more comfortable for us if data files is located in user.home instead long path in gradle caches.